sirepo/auth/__init__.py

```diff
@@ -219,22 +219,34 @@ def cookie_cleaner(self, values):
         values[_COOKIE_STATE] = _STATE_LOGGED_OUT
         return values
 
-    def create_user(self, module, want_login=False):
+    def create_user(self, module_or_method, want_login=False):
+        m = _METHOD_MODULES.get(module_or_method, module_or_method)
         u = simulation_db.user_create()
         if want_login:
-            self._login_user(module, u)
-            self._create_roles_for_new_user(module.AUTH_METHOD)
+            self._login_user(m, u)
+            self._create_roles_for_new_user(m.AUTH_METHOD)
```

Comment:
m.AUTH_METHOD won't work if m is a method and not a module.

Reply:
Was a mess, made private 695c9ef3f83ac271fecd1f4f45f84129ddb26239